### ISSUE TITLE
parser_error false positive, select with arithmetic operation

### ISSUE THREAD

**Issue by g-back** [abaplint/abaplint#786]
```abap
SELECT
    datab,
    datbi,
    kbetr * ( -1 )
   FROM a950
   INNER JOIN konp
     ON a950~knumh = konp~knumh
   WHERE a950~kschl = @provision_cond_type
     AND vkorg = @sales_org
   INTO TABLE @tm_data.
```

**Comment by larshp** [comment]
there is a lot of SELECT stuff missing...

**Comment by larshp** [comment]
`SELECT SINGLE 1 + 1 FROM ztab INTO @DATA(sdf).`

**Comment by larshp** [comment]
https://syntax.abaplint.org/?filter=sqlsour#/expression/SQLFieldList

**Comment by larshp** [comment]
first steps in https://github.com/abaplint/abaplint/pull/2116

**Comment by larshp** [comment]
@g-back try again, latest version
